api/app/controllers/spree/api/v2/platform/translatable_resource_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
```

Comment:
I would recommend doing this via concern and not adding another layer of ancestry. It will make maintenance easier and improve code readability.

Reply:
Makes sense, agree 👍

Reply:
Note, I ended up adding the scope extension into the main scope method based on whether the model class is a translatable resource, should be cleaner like this:
https://github.com/spree/spree/blob/e63e703bd3fa8fc8e6da5dfd138120a4853d7641/api/app/controllers/spree/api/v2/resource_controller.rb#L34-L38